firmware/integration/rusefi_config.txt

```diff
@@ -552,11 +557,11 @@ float vehicleSpeedCoef;+This coefficient translates vehicle speed input frequenc
 custom can_nbc_e 4 bits, U32, @OFFSET@, [0:4], "None", "FIAT", "VAG", "MAZDA RX8", "BMW", "W202", "BMW E90", "Haltech", "VAG MQB", "Nissan VQ35", "Genesis Coupe", "Honda K", "type 12", "type 13", "type 14", "INVALID"
 can_nbc_e canNbcType;set can_mode X
 
-int canSleepPeriodMs;CANbus thread period, ms;"ms", 1, 0, 0, 1000, 2
+int canSleepPeriodMs;+CANbus thread period in ms;"ms", 1, 0, 0, 1000, 2
```

Comment:
we should double check what this actually does - I think it's just the update rate for the "rusefi verbose CAN" messages

Reply:
It's the comment that was already in there so could be obsolete. 
I was going to activate as many existing tips as possible and then ask everyone to report low quality ones.